api/index.js

```diff
@@ -0,0 +1,40 @@
+import fetch from 'node-fetch'
+
+const handler = async (request, response) => {
+  const {endpoint} = request.query
```

Comment:
Why pass the endpoint? Are there multiple ones?

Also this is not safe. I’d rather have a `environment` search params that allows to pick which endpoint if there are multiple ones (or something like that).

Reply:
No there's no multiple ones indeed